packages/editor/src/pages/Editor/components/Main/Header/index.tsx

```diff
@@ -137,20 +125,27 @@ class Header extends Component<IProps, IState> {
 
     return (
       <>
-        <CommonHeader
-          items={items.map((item: IHeaderItem) => {
-            const renderedItem = this.renderItem(item);
-            if (item.key === 'solution-name') {
-              return {
-                ...renderedItem,
-                onClick: this.showSolutionSettings,
-              };
-            } else {
-              return renderedItem;
-            }
-          })}
-          farItems={farItems.map((item: IHeaderItem) => this.renderItem(item))}
-        />
+        <CopyableToClipboard
```

Comment:
this feels reaaaaaaaally strange... 
the connection between the parent and child looks magical here, and I wouldn't expect any other dev to know that there's a `.export-snippet-to-clipboard` class on one of the buttons inside a contextual menu inside the header/selectors.ts.

also, hierarchy-ly, it seems really strange for this header to be wrapped in a component that all it does is set up a clipboard with an event listener on a button inside a context menu inside of the component. 

at the very least, I think it needs a comment.